SparkPost/2web2ui
# SA-1820 - (UI) "Per precision" on unique metrics acting weird

[SA-1820](https://sparkpost.atlassian.net/browse/SA-1820)

### What Changed
- In the new report builder feature, renamed `Legend` -> `ActiveMetrics` to be more consistent
- Semantically rendered a heading instead of a div for that part of the page
- Renamed a Cypress test file to match convention
- Properly grabbing `reportOptions` from context rather than the Redux Store <-- This resolved the bug

### How To Test
- Go to `/signals/analytics` with HIbana enabled
- Click on "Add Metrics"
- Check "Unique Clicks" and "Unique Confirmed Opens" and apply those metrics
- Verify the precision content renders within the tags

### To Do
